Snabble/Core/Cart/ShoppingCart.swift

```diff
@@ -334,6 +331,11 @@ extension ShoppingCart {
         return url.appendingPathComponent(self.projectId.rawValue + ".json")
     }
 
+    private var directory: String? {
+        let docDir = FileManager.default.urls(for: .documentDirectory, in: .userDomainMask)[0]
+        return docDir.path
+    }
```

Comment:
Wenn ich es richtig sehe wird hier eine Datei `$projectId.json` gespeichert. Sollen wir es entweder `$projectId-shoppingCart.json` oder in einem Subfolder `shoppingCarts` speichern? So ist der Kontext klar und wir erzeugen potentiell keine Konflikte in der Zukunft?

Reply:
Das sollte IMO ein separater PR werden, weil dazu auch noch Migrationscode erforderlich ist.

Reply:
Stimmt 🤦 

Es fällt mir dadurch aber noch etwas anderes ein. Sollte die Configuration wie es vorher war nicht weiterhin verfügbar sein? Warum hast du sie entfernt?

Reply:
Ich hab's entfernt weil es a) niemand gebraucht hat und weil b) es sonst ohne grössere Änderungen nicht möglich war diese separat serialisierte Instanz aus dem Inflight-State wieder in's richtige Directory zu schreiben

Reply:
Problem an der Geschichte ist, das wir etwas in einen Ordner schreiben der im Kontext einer 3rd-Party App steht ohne das es ein spezifischer Snabble-Order oder mit einem Pre-/Suffix versehen ist. Ich sehe dort die Gefahr der Kollision vor allem wenn der Anwender keine Möglichkeit hat einzugreifen.

Reply:
Android verwendet einen Subfolder "Snabble"